### DISCUSSION TITLE
please add opportunity to use any external APIs with configuration without problems with cors

### DISCUSSION THREAD

**Discussion by AnoirsGit** [sveltejs/kit#9570] [Ideas]
please add opportunity to use any external APIs with configuration without problems with cors

**Comment by teemingc** [comment]
Are you encountering cors errors with your own external apis or do they belong to someone else?

**Reply by AnoirsGit** [reply]
someone else

**Reply by teemingc** [reply]
In that case, the other party is responsible for configuring their server to return the correct CORS headers. It cannot be modified by SvelteKit.

**Reply by AnoirsGit** [reply]
thanks a lot
